src/plugins/sockets/inGameEvents.js

```diff
@@ -193,7 +193,8 @@ export async function handleInGameEvents(evData) {
       // Show OpponentRequestedStalemateDialog if opponent requested stalemate
       //   and game is not yet over
       gameStore.consideringOpponentStalemateRequest =
-        !evData.victory.gameOver && evData.requestedByPNum !== gameStore.myPNum;
+        !evData.victory.gameOver && evData.playedBy !== gameStore.myPNum;
+      gameStore.waitingForOpponentToStalemate = !evData.victory.gameOver && evData.playedBy === gameStore.myPNum;
```

Comment:
Setting this now so you can see the overlay w/ waiting for your opponent on page refresh (previously it would disappear)